dev/com.ibm.websphere.appserver.mpConfig1.2-cdi1.2/com.ibm.websphere.appserver.mpConfig1.2-cdi1.2.feature

```diff
@@ -0,0 +1,11 @@
+-include= ~../cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.mpConfig1.2-cdi1.2
```

Comment:
Can you remove the cdi version from the symbolic name?

Reply:
OK, will do in a follow up PR